src/modules/append_of_likes.js

```diff
@@ -0,0 +1,16 @@
+const appendNoOFLike = (id, likeNumber) => {
+  const likeHtml = document.querySelector(`#likes-${id}`);
+  likeHtml.innerHTML = `${likeNumber} likes`;
```

Comment:
It is more readable if you sum the code like this :   `document.querySelector(`#likes-${id}`).innerHTML = `${likeNumber} likes`;`